integration_test/list_test.go

```diff
@@ -40,5 +40,41 @@ func TestKrewList(t *testing.T) {
 	if diff := cmp.Diff(eventualList, expected); diff != "" {
 		t.Fatalf("'list' output doesn't match:\n%s", diff)
 	}
-	// TODO(ahmetb): install multiple plugins and see if the output is sorted
+
+	test.Krew("uninstall", validPlugin).RunOrFail()
+	expected = []byte{'\n'}
+
+	uninstallList := test.Krew("list").RunOrFailOutput()
+	if diff := cmp.Diff(uninstallList, expected); diff != "" {
+		t.Fatalf("'list' output doesn't match:\n%s", diff)
+	}
+
+	test.Krew("install", validPlugin5, validPlugin4, validPlugin3, validPlugin2, validPlugin).RunOrFail()
+	expected = []byte(validPlugin + "\n" + validPlugin2 + "\n" + validPlugin5 + "\n" + validPlugin4 + "\n" + validPlugin3 + "\n")
+
+	sortedList := test.Krew("list").RunOrFailOutput()
+	if diff := cmp.Diff(sortedList, expected); diff != "" {
+		t.Fatalf("'list' output doesn't match:\n%s", diff)
+	}
+
+	genSpaces := func(s string) string {
```

Comment:
This `genSpaces` hack is not necessary IMHO. Why not just use a reasonable regexp?

Reply:
At the time I expected a reasonable regex to end up being more complicated; i'll swap it out.